<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1127</width>
    <height>900</height>
   </rect>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>16777215</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>10</y>
      <width>1000</width>
      <height>600</height>
     </rect>
    </property>
    <property name="maximumSize">
     <size>
      <width>1122</width>
      <height>612</height>
     </size>
    </property>
    <property name="acceptDrops">
     <bool>true</bool>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="sceneRect">
     <rectf>
      <x>0.000000000000000</x>
      <y>0.000000000000000</y>
      <width>500.000000000000000</width>
      <height>250.000000000000000</height>
     </rectf>
    </property>
   </widget>
   <widget class="QSlider" name="xSlider">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>620</y>
      <width>160</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="singleStep">
     <number>1</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="ySlider">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>620</y>
      <width>151</width>
      <height>22</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>25</number>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="xValue">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>620</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>X Achse</string>
    </property>
   </widget>
   <widget class="QLabel" name="yValue">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>620</y>
      <width>47</width>
      <height>14</height>
     </rect>
    </property>
    <property name="text">
     <string>Y Achse</string>
    </property>
   </widget>
   <widget class="QListWidget" name="shapeList">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>101</width>
      <height>601</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="dellAllButton">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>660</y>
      <width>211</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Delete all</string>
    </property>
   </widget>
   <widget class="QPushButton" name="CodeGenButt">
    <property name="enabled">
     <bool>false</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>660</y>
      <width>231</width>
      <height>51</height>
     </rect>
    </property>
    <property name="text">
     <string>Code Genrator</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rBshape">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>630</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>hei√üplatte Einrichten</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="rbGrid">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>630</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Grid Einstellen</string>
    </property>
    <property name="checked">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1127</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Shape"/>
    <addaction name="actionLoad"/>
    <addaction name="actionsave"/>
    <addaction name="actionEdit_shape"/>
    <addaction name="actionDelet_shape"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionUser_Manuell"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Shape"/>
   <addaction name="actionEdit_shape"/>
   <addaction name="actionsave"/>
   <addaction name="actionLoad"/>
   <addaction name="actionDelet_shape"/>
   <addaction name="separator"/>
   <addaction name="actionUser_Manuell"/>
  </widget>
  <action name="actionEdit_shape">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/recIcon /icon/Edit.png</normaloff>:/recIcon /icon/Edit.png</iconset>
   </property>
   <property name="text">
    <string>Edit shape</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/recIcon /icon/exit.png</normaloff>:/recIcon /icon/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionUser_Manuell">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/recIcon /icon/Doc.png</normaloff>:/recIcon /icon/Doc.png</iconset>
   </property>
   <property name="text">
    <string>User Manuell</string>
   </property>
  </action>
  <action name="actionDelet_shape">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/recIcon /icon/delete.png</normaloff>:/recIcon /icon/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delet shape</string>
   </property>
  </action>
  <action name="actionNew_Shape">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/recIcon /icon/new.png</normaloff>:/recIcon /icon/new.png</iconset>
   </property>
   <property name="text">
    <string>New Shape</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/recIcon /icon/load.png</normaloff>:/recIcon /icon/load.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
   <property name="toolTip">
    <string>Load</string>
   </property>
  </action>
  <action name="actionsave">
   <property name="icon">
    <iconset resource="Resource.qrc">
     <normaloff>:/recIcon /icon/save.png</normaloff>:/recIcon /icon/save.png</iconset>
   </property>
   <property name="text">
    <string>Save </string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="Resource.qrc"/>
 </resources>
 <connections/>
</ui>
